ladybug/src/main/java/nl/nn/ibistesttool/MessageCapturer.java

```diff
@@ -71,7 +71,7 @@ public StreamingType getStreamingType(Object message) {
 	}
 
 	@Override
-	public <T> T toOutputStream(T message, OutputStream outputStream, Consumer<String> charsetNotifier) {
+	public <T> T toOutputStream(T message, OutputStream outputStream, Consumer<String> charsetNotifier, Consumer<Throwable> exceptionNotifier) {
```

Comment:
Moet die exceptionNotifier dan niet ook aangeroepen worden als er een exceptie optreedt?

Reply:
Idealieter wel maar ik dacht dat we in 7.6 streams nog niet ondersteunden / dat het standaard uit staat en niet aanbevolen is om het aan te zetten. Zolang streams niet worden gebruikt wordt dit niet aangeroepen zover ik weet. Maar dat kun jij beter inschatten.